app/Resources/translations/messages.en_GB.xliff

```diff
@@ -437,21 +437,11 @@ For all devices with a USB port.</target>
         <target>Continue</target>
         <jms:reference-file line="37">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/displayVettingTypes.html.twig</jms:reference-file>
       </trans-unit>
-      <trans-unit id="fca84af7a11d99647b905a1af04f3dc65cd896c9" resname="ss.registration.vetting_type.button.remote_vetting">
-        <source>ss.registration.vetting_type.button.remote_vetting</source>
-        <target>Continue with remote vetting</target>
-        <jms:reference-file line="26">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/displayVettingTypes.html.twig</jms:reference-file>
-      </trans-unit>
       <trans-unit id="36f60e5402c0f9f29d5167f5448cf08e9b309176" resname="ss.registration.vetting_type.description.ra_vetting">
         <source>ss.registration.vetting_type.description.ra_vetting</source>
         <target>Vet your second factor token at one of your registration authority locations.</target>
         <jms:reference-file line="35">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/displayVettingTypes.html.twig</jms:reference-file>
       </trans-unit>
-      <trans-unit id="0f368e2c1b03805b28c1ce11ecd3db8358024386" resname="ss.registration.vetting_type.description.remote_vetting">
-        <source>ss.registration.vetting_type.description.remote_vetting</source>
-        <target>Prove your identity using one of three verification methods. ReadId document scanner (read your passport, drivers licence or ID card using the ReadId connect app and a NFF compatible document), IRMA (RMA offers a way for privacy-friendly authentication. When authenticating, you as a user reveal only relevant properties (attributes) of yourself, using an IRMA app on your mobile phone — for instance that you are older than 16.) or iDIN (bank).</target>
```

Comment:
Removing this information is a regression in my opinion. Either add this text to the existing text, or add the paragraph underneath the existing page introduction text.

Reply:
Added a paragraph with this information above the blocks.

Reply:
Thanks!